lib/controllers/root-controller.js

```diff
@@ -426,6 +386,49 @@ export default class RootController extends React.Component {
     );
   }
 
+  renderFilePatches() {
+    if (this.props.filePatchItems.length === 0) {
+      return null;
+    }
+
+    return this.props.filePatchItems.map(item => {
+      const {filePath, stagingStatus, workdir} = getPropsFromUri(item.uri);
+      const repository = this.props.getRepositoryForWorkdir(workdir);
+      const state = this.repositoryStateRegistry.getStateForModel(repository);
+      const amending = state.amending;
+
+      return (
+        <PaneItem
+          key={item.key}
+          workspace={this.props.workspace}
+          ref={c => { this.filePatchControllerPane = c; }}
```

Comment:
Do we use this ref for anything? Each item rendered in the array will overwrite the property.